README.md

```diff
@@ -83,19 +93,31 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+es5
+no
+obj(3, 6
 
 ## What does the following code output to the console?
 
 ```js
 var i = 0
-for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') ) { 
+for ( console.log('Bob'); (console.log('Curly') === undefined); console.log('Mo') ) { 
   console.log(++i)
   if (i >= 2)
     break
 }
 console.log(i)
 ```
-
--- YOUR ANSWER HERE --
-
+(bob,
+curly,
+1,
+mo,
+curly,
+2,
+2,
+)
+bob
+curly
+Mo
+1
+0
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please spend some time playing with for loops like Carlos did in class